.github/workflows/ci.yml

```diff
@@ -961,7 +961,10 @@ ${{ hashFiles('.pre-commit-config.yaml') }}"
         run: ./scripts/ci/testing/run_downgrade_test.sh
       - name: "Test Offline SQL generation"
         run: ./scripts/ci/testing/run_offline_sql_test.sh
-      - name: "Tests: ${{needs.build-info.outputs.test-types}}"
+      - name: "Tests: ${{needs.build-info.outputs.testTypes}}"
+        run: breeze tests --run-in-parallel
+        env:
+          PYTHON_VERSIONS: ${{ needs.build-info.outputs.pythonVersionsListAsString }}
         run: ./scripts/ci/testing/ci_run_airflow_testing.sh
```

Comment:
The all "ci_run_airflow_testing.sh" run shoudl be removed - I think this what makes it not run currently becausse there is eror when parsing the yml file.